packages/amplication-client/src/Purchase/FAQ.tsx

```diff
@@ -19,115 +19,118 @@ export const FAQ = () => {
-                subject="Amplication Enterprise Plan Inquiry"
-                body="contact sales"
-              />{" "}
-              for more information and to purchase.
+              Amplication allows you to deploy to AWS's Elastic Container
+              Service (ECS). We provide in-depth documentation that will walk
+              you through using various Amplication plugins to provision the
+              required infrastructure and set up a process for continuous
+              integration and deployment.
+              <br />
+              <br />
+              The steps cover manual configurations for permissions and
+              authentication, plugin settings for AWS services, code generation,
+              provisioning with Terraform, and deployment using GitHub Actions.
             </div>
           }
         />
         <Question
-          question="We're an open-source project, is there special pricing available?"
+          question="I have an existing DB with a lot of data, entities, tables, and relationships. Can I use your backend development platform, while still keeping the data and the knowledge?"
+          answer="Amplication's DB Schema Import feature provides a clear pathway for integrating an existing database with the Amplication platform. The import feature is available on all Amplication plans, including Free accounts."
+        />
+        <Question
+          question="Will your generated code override my code changes?"
           answer={
             <div>
-              Yes, as part of our commitment to support open-source projects, we
-              offer the pro plan for free. Reach out{" "}
-              <MailTo
-                email="mailto:sales@amplication.com"
-                subject="Pro plan for Open Source project"
-                body="here"
-              />{" "}
-              and we will set you up.
+              Amplication's Smart Git Sync feature ensures that changes made in
+              an Amplication project are automatically tracked and committed to
+              a dedicated branch in the connected Git repository. The system is
+              designed to manage the syncing process efficiently, automatically
+              creating commits and pull requests with clear messages that
+              reflect the changes made.
+              <br />
+              <br />
+              Your changes will not get overriden until you have merged any new
+              changes from Amplication.
             </div>
           }
         />
+        <Question
+          question="Due to security regulations, my enterprise require supporting relevant capabilities, like 2FA, audit log, SSO and more. Do you support it?"
+          answer="The Enterprise edition gives you access to advanced security features like SSO and audit logs are provided."
+        />
+        <Question
+          question="Can I pay to Amplication through my AWS account (AWS billing)?"
+          answer=""
```

Comment:
The answer should be something like- Yes, please contact us to handle it (or such).